src/components/forms/NumberField/_input-container.scss

```
@@ -0,0 +1,77 @@
+@use '@/scss/bem';
+
+// Note: this was ported from the SDK
+
+$affixes: 'prefix', 'suffix';
+
+@mixin override($property, $modifier) {
+  $default: var(--of-input-container-affix-#{$property});
+  #{$property}: var(--of-input-container-#{$modifier}-#{$property}, #{$default});
+}
+
+.#{'openforms-input-container'} {
```

this essentially makes up its own component `InputContainer` and the styling code + wrapper JSX probably belong in their own place in `src/components/forms`. They can be applied in a broader context than just number fields.

Other than that, this is a bit of an odd construct in SASS, easier to understand and read is:


```suggestion
.openforms-input-container {
```